test/jdk/javax/swing/JScrollBar/4865918/bug4865918.java

```diff
@@ -73,8 +84,8 @@ private static int getValue() throws Exception {
         return result[0];
     }
 
-    private static void setupTest() {
-
+    private static void createAndShowGUI() {
+        frame = new JFrame("bug4865918");
```

Comment:
`frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);` was set in the initial test but not now.
Is it required to restore?

Also there was a delay of 200ms after waiting for timeout, that is also not restored.